### DISCUSSION TITLE
Cannot download Windows Zip file

### DISCUSSION THREAD

**Discussion by Pewit35** [vzhd1701/enex2notion#103] [General]
I have tried Chrome and MS Edge and both report network errors when trying to download the Windows Zip file. (bin_enex2notion_0.3.1_win_x64.zip)
Other files have no problem downloading.

**Comment by vzhd1701** [comment]
Works OK on my side. What error are you getting?

**Reply by Pewit35** [reply]
"Stalled" In the network log

**Reply by vzhd1701** [reply]
Maybe it's getting blocked by your antivirus. Check detection log and quarantine.
